graylog2-server/src/main/java/org/graylog/events/notifications/TemplateModelProvider.java

```diff
@@ -52,7 +58,7 @@ public Map<String, Object> of(EventNotificationContext ctx, List<MessageSummary>
         return ImmutableMap.<String, Object>builder()
                 .putAll(customFields)
                 .putAll(objectMapperProvider.getForTimeZone(timeZone).convertValue(modelData, TypeReferences.MAP_STRING_OBJECT))
-                .put("http_external_uri", this.httpExternalUri)
+                .put("http_external_uri", isDevelopmentServer ? URI.create("http://localhost:8080/") : this.httpExternalUri)
```

Comment:
I don't know that we need to change this, or at least that we need to hardcode it so completely. Maybe someone runs their dev server on port 8082, for example. If we do want to modify this code to work better in development environments, I think we can leave it for a smaller, separate PR.

Reply:
Fair enough, this was mostly just to enable testing the generated links but I'll remove it from this PR before merging.